<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlgSuperficiePoligonal</class>
 <widget class="QDialog" name="dlgSuperficiePoligonal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>280</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Polygonal surface</string>
  </property>
  <property name="windowIcon">
   <iconset resource="recursos.qrc">
    <normaloff>:/dlg/dock/PolygonalArea</normaloff>:/dlg/dock/PolygonalArea</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Enter coordinates clockwise.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="tblPoints">
     <property name="contextMenuPolicy">
      <enum>Qt::CustomContextMenu</enum>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="cornerButtonEnabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Calculation</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="lblFormula">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
  <action name="actionAddPoint">
   <property name="text">
    <string>Add Coordinate</string>
   </property>
   <property name="toolTip">
    <string>Agrega una coordenada a la lista</string>
   </property>
  </action>
  <action name="actionDelete_Coordinate">
   <property name="text">
    <string>Delete Coordinate</string>
   </property>
   <property name="toolTip">
    <string>Borra la coordenada seleccionada</string>
   </property>
  </action>
  <action name="actionMove_Up">
   <property name="text">
    <string>Move Up</string>
   </property>
  </action>
  <action name="actionMove_Down">
   <property name="text">
    <string>Move Down</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="recursos.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>dlgSuperficiePoligonal</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>dlgSuperficiePoligonal</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
